Mathlib/Data/Multiset/Basic.lean

```diff
@@ -2674,6 +2680,9 @@ theorem add_eq_union_iff_disjoint [DecidableEq α] {s t : Multiset α} :
   simp_rw [← inter_eq_zero_iff_disjoint, ext, count_add, count_union, count_inter, count_zero,
     Nat.min_eq_zero_iff, Nat.add_eq_max_iff]
 
+theorem add_eq_sup_iff_disjoint [DecidableEq α] {s t : Multiset α} :
+    s + t = s ⊔ t ↔ Disjoint s t := by rw [← add_eq_union_iff_disjoint, sup_eq_union]
+
```

Comment:
```suggestion
```
I don't think we will ever need this as `add_eq_sup_iff_disjoint` is equivalent to `add_eq_union_iff_disjoint` due to `sup_eq_union`.